src/Command/ImportVotePlacesCommand.php

```diff
@@ -0,0 +1,116 @@
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\VotePlace;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
+
+class ImportVotePlacesCommand extends Command
+{
+    private const BATCH_SIZE = 1000;
+
+    private $em;
+
+    public function __construct(EntityManagerInterface $em)
+    {
+        $this->em = $em;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+          ->setName('app:import:vote-places')
+          ->addArgument('fileUrl', InputArgument::REQUIRED)
+          ->setDescription(
+            'Import vote places from file store in Google Storage'
+          )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $rows = $this->parseCSV($input->getArgument('fileUrl'), 'r');
+        } catch (FileNotFoundException $exception) {
+            $output->writeln(
+              sprintf('%s file not found', $input->getArgument('fileUrl'))
+            );
+
+            return 1;
+        }
+
+        $this->em->beginTransaction();
+
+        $this->createAndPersistVotePlaces($rows);
+
+        $this->em->flush();
+        $this->em->commit();
+
+        $output->writeln(['', 'Vote places load OK']);
+    }
+
+    private function parseCSV(string $filename): array
+    {
+        $rows = [];
+        if (false === ($handle = fopen($filename, 'r'))) {
+            throw new FileNotFoundException(sprintf('% not found', $filename));
+        }
+
+        while (false !== ($data = fgetcsv($handle, 10000, ','))) {
+            $row = array_map('trim', $data);
+
+            if ('nom_region' === $row[0]) {
+                continue;
+            }
+
+            $rows[] = [
+                'code_postal' => $row[4],
+                'nom_commune_bdv' => $row[6],
+                'code_bdv' => $row[8],
+                'nom_bdv' => $row[9],
+                'adresse_bdv' => $row[10],
+            ];
+        }
+
+        fclose($handle);
+
+        return $rows;
+    }
+
+    private function createAndPersistVotePlaces(array $rows): void
+    {
+        $batch = 0;
+
+        foreach ($rows as $row) {
+            if ($this->em->getRepository(VotePlace::class)->findOneByCode($row['code_bdv'])) {
+                continue;
+            }
+
+            $votePlace = new VotePlace();
+            $votePlace->setName($row['nom_bdv']);
+            $votePlace->setAddress($row['adresse_bdv']);
+            $votePlace->setPostalCode(false !== strpos($row['code_postal'], '/')
+                ? str_replace('/', ',', $row['code_postal'])
+                : $row['code_postal']
+            );
+            $votePlace->setCity($row['nom_commune_bdv']);
```

Comment:
You save city name to `city_insee` column. https://github.com/EnMarche/en-marche.fr/pull/3502/files#diff-ede16518c28d1976c28cefba48b2c9bfR72
Is the column name wrong or the data you set?